crates/ruma-client-api/src/r0/room/create_room.rs

```diff
@@ -129,27 +129,31 @@ pub struct CreationContent {
     /// A reference to the room this room replaces, if the previous room was upgraded.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub predecessor: Option<PreviousRoom>,
+
+    /// The room type that shall be set. This is currently used for spaces
+    #[serde(skip_serializing_if = "Option::is_none", rename = "type")]
+    pub room_type: Option<String>,
```

Comment:
Why is this a `String` when you added a dedicated `RoomType` type?

Reply:
Oh that's I think a typo :o will fix 😅

Reply:
Fixed in 5fac97ae0653260c307b7779259fc95cb6f08d75